### PR TITLE
Add changelog for x86_omf

### PR DESCRIPTION
**Pull request by shoo**
A changelog for https://github.com/dlang/dub/pull/1983

### CODE REVIEW

File: changelog/add_x86_omf.dd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Added architecture x86_omf for windows-dmd
+
+In windows dmd, an architecture x86_omf is enabled for
```
Comment (thewilsonator) [inline changelog/add_x86_omf.dd:3]: `_On_ windows dmd, an architecture x86_omf is enabled for`

File: changelog/add_x86_omf.dd [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Added architecture x86_omf for windows-dmd
+
+In windows dmd, an architecture x86_omf is enabled for
+the platform specs when explicitly specify --arch=x86_omf or --arch=x86.
```
Comment (thewilsonator) [inline changelog/add_x86_omf.dd:4]: `the platform specs when explicitly specif_ied_ via --arch=x86_omf or --arch=x86.`

Also, no newline at the end of file

File: changelog/add_x86_omf.dd (hunk 1 above)
Comment (WebFreak001) [reply changelog/add_x86_omf.dd:3]: how about rewriting the whole sentence a little like
```
On windows dmd the x86_omf architecture can now be used as platform
specification using `--arch=x86_omf`. This is currently equivalent to
the previously existing `--arch=x86` argument.
```

Comment (thewilsonator) [reply changelog/add_x86_omf.dd:3]: That is much better, thanks.

**Comment by shoo** [comment]
Thank you for proofreading the text.

Please also see the following document changes:
https://github.com/dlang/dub-docs/pull/30

**Comment by shoo** [comment]
What else is there to do with this PR?

**Pull request merged**